### PR TITLE
Add turtlebot3_description in robot workspace

### PR DESCRIPTION
**Pull request by ahtsan**
Issue #, if available:
Currently, robot model is not visible in rviz because the parameter robot_description hasn't been created.

Description of changes:
This PR adds turtlebot3_description as a dependency in robot workspace, and create the parameter robot_description correctly in rotate.launch

Tested both locally and on AWS Robomaker.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by mm318 — APPROVED** [review]
I have the same comment here as https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/pull/70#pullrequestreview-471032219.

File: robot_ws/src/cloudwatch_robot/package.xml [hunk 1]
```diff
@@ -35,4 +35,6 @@
 
   <!-- Turtlebot -->
   <depend>turtlebot3_msgs</depend>
+  <!-- Required to install .rviz model-->
+  <depend>turtlebot3_description</depend>
```
Comment (mm318) [inline robot_ws/src/cloudwatch_robot/package.xml:39]: This hasn't been changed to `turtlebot3_description_reduced_mesh`.

**Review by mm318 — CHANGES_REQUESTED** [review]

File: robot_ws/src/cloudwatch_robot/package.xml (hunk 1 above)
Comment (ahtsan) [reply robot_ws/src/cloudwatch_robot/package.xml:39]: turtlebot3_description is needed for the rviz, so we still need it. And yes, we can add turtlebot3_description_reduced_mesh as dependency as well, and do the same for hello world sample app.

Comment (ahtsan) [reply robot_ws/src/cloudwatch_robot/package.xml:39]: @mm318 Added `turtlebot3_description_reduced_mesh` as a dependency. Please take a look.

**Review by mm318 — APPROVED** [review]

**Pull request merged**